contracts/src/Fees.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+library PDPFees {
+
+    // Estimate of USD price in FIL
+    uint256 constant FIL_USD_NUM = 10;
+    uint256 constant FIL_USD_DENOM = 35;
```

Comment:
Numerator/denominator seem backwards. If these really are the intended numbers, please explain in a comment (perhaps do so anyway).